server/src/main/java/server/BazelServices.java

```diff
@@ -12,31 +12,38 @@
 import org.eclipse.lsp4j.services.LanguageClientAware;
 import org.eclipse.lsp4j.services.TextDocumentService;
 import org.eclipse.lsp4j.services.WorkspaceService;
+import server.util.DocumentTracker;
```

Comment:
Do we want to separate our import statements distinguish third part/local packages?

```
import third.party.package1;
import third.party.package2;
import third.party.package3;

import local.package1;
import local.package2;
import local.package3;
```

Reply:
Maybe we can add this as part of the java styling package thing